obspy/clients/fdsn/tests/test_client.py

```diff
@@ -847,7 +848,7 @@ def custom_side_effects(*args, **kwargs):
         # Some custom urls
         base_url_event = "http://other_url.com/beta/event_service/11"
         base_url_station = "http://some_url.com/beta2/station/7"
-        base_url_ds = "http://new.com/beta3/dataselect/8"
+        base_url_ds = "http://new_url.com/beta3/dataselect/8"
```

Comment:
I did not notice these before, but they are valid (if unused at the moment) domain names. It might be better to use `example.{com,org,edu}` or `something.example` which are reserved.